src/mcts/node.h

```diff
@@ -79,18 +79,16 @@ class Node {
   // Returns p / N, which is equal to U / (cpuct * sqrt(N[parent])) by the MCTS
   // equation. So it's really more of a "reduced U" than raw U.
   float GetU() const { return p_ / (1 + n_ + n_in_flight_); }
-  // Returns value of Value Head returned from the neural net.
-  float GetV() const { return v_; }
   // Returns value of Move probability returned from the neural net
   // (but can be changed by adding Dirichlet noise).
   float GetP() const { return p_; }
   // Returns whether the node is known to be draw/lose/win.
   bool IsTerminal() const { return is_terminal_; }
+  // Unlike GetQ, this is works when n_ == 0
```

Comment:
It's better to write what it does, not only how it differs. Also, "is" is not needed. Also, period in the end. :)

Also, maybe it's better to remove this function and make GetQ() check for `is_terminal_`?

Reply:
You're the one who added the function :P I kinda like having a separate function for terminal nodes though, it may help prevent human errors regarding conditionals like `if (node->IsTerminal())`.